From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Fri, 10 Feb 2017 09:13:40 +0100
Subject: [PATCH] PVE: [Config] lxc.service: start after a potential
 syslog.service

We could add this as a snippet from pve-container instead.

Signed-off-by: Wolfgang Bumiller <w.bumiller@proxmox.com>
---
 config/init/systemd/lxc.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/init/systemd/lxc.service.in b/config/init/systemd/lxc.service.in
index 397a6c4d0..6924374d9 100644
--- a/config/init/systemd/lxc.service.in
+++ b/config/init/systemd/lxc.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=LXC Container Initialization and Autoboot Code
-After=network.target lxc-net.service remote-fs.target
+After=network.target lxc-net.service remote-fs.target syslog.target
 Wants=lxc-net.service
 Documentation=man:lxc-autostart man:lxc
 
